src/Norm/base/UIUnit.cpp

```diff
@@ -224,8 +226,16 @@ void UIUnit::Open()
 					}
 				}
 
-				// Arret sinon
-				commandFile.AddInputCommandFileError("Incorrect command: " + sCommand + " " + sValue);
+				// Arret sinon, avec message d'erreur comportant la commande complete
+				sFullCommand = "";
+				for (j = 0; j < svIdentifierPath.GetSize(); j++)
+				{
+					if (j > 0)
+						sFullCommand += ".";
```

Comment:
(Stylistic remark) I would have either:
- surrounded this instruction with curly braces, to emphasize that this is the scope of the `if` above (and thus facilitate debugging via temporary, ad-hoc, `std::cout << ` insertions;
- put this instruction on the same line as the `if` above, to emphasize that this is a single line with no debugging envisaged.

Reply:
Ce pattern de boucle avec un séparateur uniquement à partir du deuxième élément est particulier. Il intervient très souvent, et le garder avec une syntaxe très légère rend le code moins verbeux et plus lisible.

Sinon, je suis d'accord en général avec l'argument de la facilité de debugging. Dans bien d'autres cas, j'utilise alors les accolades, même pour des bloc if mono-instructions.

Par contre, il n'est pas possible de mettre l'instruction sur la même ligne que le if: c'est le pretty-printer du C++ qui se charge de la mise en forme?

Reply:
OK, I got your point. And OK for the linter part. Hence, I'll close this conversation thread.